base/strings/basic.jl

```diff
@@ -34,6 +34,7 @@ If you need to subsequently modify `v`, use `String(copy(v))` instead.
 """
 String(v::Array{UInt8,1})
 
+String(x::Symbol) = string(x)
```

Comment:
I'd prefer a single PR.  Ideally @dbeach24 would update this PR, but he doesn't seem to be around so maybe we should fork his branch and push a new PR?